dcos/subcommand.py

```diff
@@ -230,20 +303,31 @@ def _install_env(pkg, options):
     """
 
     pkg_dir = _package_dir(pkg.name())
+    resources = pkg.resource_json()
+    binary_cli = _get_cli_binary(resources)
 
-    install_operation = pkg.command_json()
-
-    env_dir = os.path.join(pkg_dir,
-                           constants.DCOS_SUBCOMMAND_VIRTUALENV_SUBDIR)
-
-    if 'pip' in install_operation:
-        _install_with_pip(
+    if binary_cli:
+        _install_with_binary(
             pkg.name(),
-            env_dir,
-            install_operation['pip'])
+            pkg_dir,
+            binary_cli)
```

Comment:
I haven't looked at the implementation but why does this take `pkg_dir` and `_install_with_pip` take `env_dir`?